### ISSUE TITLE
BelongsTo not referencing table names properly

### ISSUE THREAD

**Issue by charlielangridge** [laravel/nova-issues#289]
I've setup a pivot table with a non standard name, that works properly through tinker, but when I add the relationship as a field in Nova it gives me a 'Class 'App\Nova\Contact' not found' error

App\Models\Event.php
`
 public function contacts()
    {
        return $this->belongsToMany('App\User', 'event_contacts', 'event_id', 'user_id');
    }
`

App\User.php
`
public function contact_for_events()
    {
        return $this->belongsToMany('App\Models\Event', 'event_contacts', 'user_id', 'event_id');
    }
`

App\Nova\User.php
`
BelongsToMany::make('Contact for Events', 'contact_for_events')
`

App\Nova\Event.php
`
BelongsToMany::make('Contacts')
`

**Comment by kenneth-g-flux** [comment]
Try adding your Nova resource as third parameter:

App\Nova\User.php
```
BelongsTo::make('Contact for Events', 'contact_for_events', App\Nova\Event::class);
```
App\Nova\Event.php
```
BelongsTo::make('Contacts', 'contacts', App\Nova\User::class);
```

**Comment by charlielangridge** [comment]
Ok, that sort of works - now the User relation shows at the bottom of the view for the event, but if i go to attach a user i get "Trying to get property 'resourceClass' of non-object"

**Comment by kenneth-g-flux** [comment]
That might have something to do with issue https://github.com/laravel/nova-issues/issues/76, have a look there.

**Comment by meyer59** [comment]
Should be fixed in Nova v1.0.8 check it out
